src/icons/Reward.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react'
+import Svg, { Path } from 'react-native-svg'
+
+const Reward = ({ size = 34 }: { size?: number } = {}) => (
+  <Svg width={size} height={size} viewBox={`0 0 ${size} ${size}`} fill="none">
```

Comment:
I don't believe this will work correctly if you specify different size params. Here's some useful info on how viewbox / width / height work https://valora-app.slack.com/archives/C025V1D6F3J/p1724705958982739